browser/ui/webui/brave_wallet/android/android_wallet_page_ui.cc

```diff
@@ -52,11 +52,9 @@ AndroidWalletPageUI::AndroidWalletPageUI(content::WebUI* web_ui,
 
   // Add required resources.
   if (url.host() == kWalletPageHost) {
-    webui::SetupWebUIDataSource(
-        source,
-        UNSAFE_TODO(base::make_span(kBraveWalletPageGenerated,
-                                    kBraveWalletPageGeneratedSize)),
-        IDR_WALLET_PAGE_HTML);
+    webui::SetupWebUIDataSource(source,
+                                base::make_span(kBraveWalletPageGenerated),
```

Comment:
This and all the other cases should be just `base::span`, and rely on CTAD. I say *should* because there could be a rare case where you still may need `base::make_span` [until this CL lands](https://chromium-review.googlesource.com/c/chromium/src/+/5962112) through 132, but I think it would be good if we replaced `base::make_span` with `base:span` wherever it is possible to.

Reply:
fixed